### PR TITLE
CDAP-13260 Metadata CLI docs updates [Part 3]

### PR DESCRIPTION
**Pull request by rohitsinha54**
### Part 3

- CLI Updates for custom metadata entities

Build: https://builds.cask.co/browse/CDAP-DQB414-1

Rendered page: https://builds.cask.co/artifact/CDAP-DQB414/shared/build-1/Docs-HTML/5.0.0-SNAPSHOT/en/reference-manual/cli-api.html#H3869

### CODE REVIEW

File: cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java [hunk 1]
```diff
@@ -173,7 +173,14 @@ public String toString() {
     "or '<namespace-name>.<stream-name>'.\n" +
     "\n" +
     "For (stream) views, '<entity-id>' includes the stream " +
-    "that they were created from: '<namespace-name>.<stream-name>.<view-name>'.";
+    "that they were created from: '<namespace-name>.<stream-name>.<view-name>'." +
+    "\n" +
+    "Custom entities can be specified as hierarchical key-value pair with an optional type if the last key in" +
```
Comment (sagarkapare) [inline cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java]: nit: add space after ` if the last key in`

**Comment by sagarkapare** [comment]
minor nit else LGTM 👍

**Comment by rohitsinha54** [comment]
@sagarkapare Address comment also fixed the text overflow in docs page. 

https://builds.cask.co/artifact/CDAP-DQB414/shared/build-3/Docs-HTML/5.0.0-SNAPSHOT/en/reference-manual/cli-api.html#H3145

**Comment by sagarkapare** [comment]
LGTM 👍

**Pull request merged**